packages/node/src/utils/cosmos.ts

```diff
@@ -105,9 +118,19 @@ export function filterEvents(
 
   const filters =
     filterOrFilters instanceof Array ? filterOrFilters : [filterOrFilters];
-  const filteredEvents = events.filter((event) => {
-    filters.find((filter) => filterEvent(event, filter));
-  });
+  const filteredEvents = events
+    .filter((event) => {
+      filters.find((filter) => filterEvent(event, filter));
+    })
+    .map((evt) => {
+      evt.msg.msg = {
+        ...evt.msg.msg,
+        ...evt.msg.msg.decodedMsg,
+      };
+      delete evt.msg.msg.decodedMsg;
```

Comment:
Is this hoisting the change from the wrappers? I think this is quite hard to follow. 

Would we be better off making a breaking change to get more usable types?